### ISSUE TITLE
UI Zoom still Working with Mouse Gesture

### ISSUE THREAD

**Issue by 3DEsprit** [decosoftware/deco-ide#159]
Even though the hotkeys are disabled, the mouse gestures still affect the UI. Definitely odd for anyone who accidentally activates this. 

![deco-ui zoomed in](https://cloud.githubusercontent.com/assets/3156278/18411260/c443c4cc-7739-11e6-8bec-6659e8fc17d6.png)

**Comment by dabbott** [comment]
Good catch. Looks like we need to call this: https://github.com/electron/electron/issues/3609
